tests/components/generic_events/aggregate/index.test.js

```diff
@@ -231,17 +233,6 @@ describe('sub-features', () => {
       targetTag: 'BUTTON',
       globalFoo: 'globalBar'
     })
-    expect(harvest[0].payload.body.ins[1]).toMatchObject({
-      eventType: 'UserAction',
-      timestamp: expect.any(Number),
-      action: 'click',
-      actionCount: 1,
-      actionDuration: 0,
-      target: 'html>body>button#myBtn:nth-of-type(2)',
-      targetId: 'myBtn',
-      targetTag: 'BUTTON',
-      globalFoo: 'globalBar'
-    })
```

Comment:
What caused this test change/deletion btw?

Reply:
probably a merge remnant.  not sure here, will check